assets/gql/contact_wa_groups/sync.gql

```diff
@@ -0,0 +1,9 @@
+mutation syncWaGroupContacts {
```

Comment:
is it `syncWaGroupContacts` or `syncContactWaGroups`, maintain consistency in naming